lib/completion-templates.ts

```diff
@@ -5,45 +5,47 @@ export const completionShTemplate = `###-begin-{{app_name}}-completions-###
 # Installation: {{app_path}} {{completion_command}} >> ~/.bashrc
 #    or {{app_path}} {{completion_command}} >> ~/.bash_profile on OSX.
 #
-_{{app_name}}_yargs_completions()
+_{{safe_app_name}}_yargs_completions()
 {
-    local cur_word args type_list
+    if [[ {{app_name_word_check}} ]]; then
```

Comment:
This line turns in to something like `if [[ "${COMP_WORDS[0]}" == "yarn" && "${COMP_WORDS[1]}" == "webpack" ]]; then` (for a script name of `yarn webpack`)